### PR TITLE
Convert to one sentence per line

### PR DESCRIPTION
**Pull request by maximiliankolb**
This PR converts the Configuring Ansible guide to follow the `one sentence per line` idea.
See #326

### CODE REVIEW

File: guides/common/modules/proc_adding-rhel-system-roles.adoc [hunk 1]
```diff
@@ -24,12 +27,15 @@ Before subscribing to the Extras channels, see the https://access.redhat.com/sup
...
 +
-The `rhel-system-roles` package downloads to `/usr/share/ansible/roles/`. You can view and make any modifications that you want to the files before you import.
+The `rhel-system-roles` package downloads to `/usr/share/ansible/roles/`.
+You can view and make any modifications that you want to the files before you import.
 
 
 . In the {Project} web UI, navigate to *Configure* > *Roles* and click the {SmartProxy} that contains the roles that you want to import.
 . From the list of Ansible roles, select the check box of the roles you want to import, and then click *Update*.
 
 You can now assign Ansible roles to hosts or host groups. For more information, see {ConfiguringAnsibleDocURL}adding-ansible-roles-to-an-existing-host_ansible[Assigning Ansible Roles to an Existing Host] in _Configuring {Project} to Use Ansible_.
```
Comment (spetrosi) [inline guides/common/modules/proc_adding-rhel-system-roles.adoc:37]: A new line is needed here.

**Review by spetrosi — CHANGES_REQUESTED** [review]
One minor correction, other than that looks good. Please fix the new line and fix merge conflicts and rebase.

File: guides/common/modules/proc_adding-rhel-system-roles.adoc (hunk 1 above)
Comment (maximiliankolb) [reply guides/common/modules/proc_adding-rhel-system-roles.adoc:37]: Fixed.

**Comment by maximiliankolb** [comment]
Fixed and rebased.

**Comment by spetrosi** [comment]
Thanks @maximiliankolb, the changes look good.

**Pull request merged**